requirements.txt

```diff
@@ -5,4 +5,5 @@ gunicorn==21.2.0
 psycopg[binary]==3.1.17
 pytest==8.0.0
 pytest-django==4.8.0
-sqlparse==0.4.4
\ No newline at end of file
+sqlparse==0.4.4
+django-handyhelpers>=0.3.14
```

Comment:
nit: typically alphabetically ordered

Reply:
I'll fix that; we should add to style guide too.